lib/presentation/Drawer/Drawer.css

```
@@ -0,0 +1,100 @@
+@import "@/styles/layers.css";
+@reference "@/styles/tokens.css";
+
+@layer components {
+  .drawer-backdrop {
+    --backdrop-opacity: 0.2;
+    --bleed: 3rem;
+    position: fixed;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)